helm/prometheus-rules/templates/alerting-rules/alertmanager.rules.yml

```diff
@@ -11,25 +11,26 @@ spec:
   groups:
   - name: alertmanager
     rules:
-    - alert: AlertManagerNotificationsFailing
+    - alert: AlertmanagerPageNotificationsFailing
```

Comment:
I think the expressions are wrong. If intégration is not opsgenie then it is a notify alert and vice versa.

Reply:
Well, no it's the opposite.
If opsgenie notification fails, that means we can't request opsgenie, so we can't page, so we notify.

Reply:
Oh yes I see your logic!
And your logic is logic....
I'm so complicated in my head...

Reply:
lool :D